activerecord/lib/active_record/transactions.rb

```diff
@@ -11,6 +11,7 @@ module Transactions
                        :before_commit_without_transaction_enrollment,
                        :commit_without_transaction_enrollment,
                        :rollback_without_transaction_enrollment,
+                       terminator: deprecated_false_terminator(ActiveRecord::Transactions),
```

Comment:
I'm actually against this change. `ActiveRecord::Transactions` would make no sense to the average user. It would be much better to just say Active Record callbacks.